tests/MongoDB.Driver.Tests/MongoCollectionImplTests.cs

```diff
@@ -1508,19 +1508,24 @@ private class ClassForDistinctWithArrayField
         public void Indexes_CreateOne_should_execute_a_CreateIndexesOperation(
             [Values(false, true)] bool usingSession,
             [Values(false, true)] bool usingCreateOneIndexOptions,
+            [Values(false, true)] bool usingWildCardIndex,
```

Comment:
Lowercase "c" in "usingWildcardIndex" for consistency?

Reply:
Done